docs/sources/installation/windows.md

```diff
@@ -43,3 +43,9 @@ Read more about the [configuration options]({{< relref "configuration.md" >}}).
 The Grafana backend includes Sqlite3 which requires GCC to compile. So
 in order to compile Grafana on Windows you need to install GCC. We
 recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
+
+## Logging in for the first time
+
+To run Grafana open your browser and go to port 3000 which is the default port. If you have changed the port you go to that port. There you will see the login page. User name is admin and password is admin. When you log in for the first time you will be asked to change your password. You can later go to user preferences and change your user name.
```

Comment:
> To run Grafana open your browser and go to port 3000

To run Grafana open your browser and go to the port you configured [above](/installation/windows/#configure), e.g. `http://localhost:8080`.

Would be really nice if each installation guideline links to the getting started page. Maybe possible then to shorten the "Logging in for the first time" for each platform and have the details in getting started. What do you think?